go.pose

```diff
@@ -98,7 +98,7 @@
 
  (entry
   (id "sicp")
-  (uri "//mitpress.mit.edu/sites/default/files/sicp/index.html")
```

Comment:
It's here.

https://mitpress.mit.edu/9780262510875/structure-and-interpretation-of-computer-programs/

It's disappointing that MIT Press can't seem to keep its URLs stable.  MIT hosts W3C, after all.